src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -475,9 +521,10 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
 
         internal static async Task ExecuteWithWatchersAsync(IFunctionInstance instance,
             IReadOnlyDictionary<string, IValueProvider> parameters,
-            TimeSpan? globalFunctionTimeout,
+            JobHostFunctionTimeoutConfiguration timeoutConfig,
```

Comment:
I'll remove it JobHost; it's a mouthful

---
In reply to: [76691087](https://github.com/Azure/azure-webjobs-sdk/pull/799#discussion_r76691087) [](ancestors = 76691087)